Analysis/PWGDQ/include/PWGDQCore/VarManager.h

```diff
@@ -544,13 +550,33 @@ void VarManager::FillTrack(T const& track, float* values)
       }
       values[kTrackDCAxy] = track.dcaXY();
       values[kTrackDCAz] = track.dcaZ();
+      if constexpr ((fillMap & ReducedTrackBarrelCov) > 0) {
+        if (fgUsedVars[kTrackDCAsigXY])
+          values[kTrackDCAsigXY] = track.dcaXY() / std::sqrt(track.cYY());
+        if (fgUsedVars[kTrackDCAsigZ])
+          values[kTrackDCAsigZ] = track.dcaZ() / std::sqrt(track.cZZ());
+        if (fgUsedVars[kTrackDCAresXY])
+          values[kTrackDCAresXY] = std::sqrt(track.cYY());
+        if (fgUsedVars[kTrackDCAresZ])
+          values[kTrackDCAresZ] = std::sqrt(track.cZZ());
+      }
     }
   }
 
   // Quantities based on the barrel track selection table
   if constexpr ((fillMap & TrackDCA) > 0) {
     values[kTrackDCAxy] = track.dcaXY();
     values[kTrackDCAz] = track.dcaZ();
+    if constexpr ((fillMap & ReducedTrackBarrelCov) > 0) {
```

Comment:
Here you should use "TrackCov" instead of the ReducedTrackBarrelCov (which is the skimmed version).

Reply:
Hi @iarsene,
i have included your changes in the newest version.
Sorry for the mix up.